### PR TITLE
Update Tracker Protocol and add example requests

### PR DESCRIPTION
**Pull request by paulboocock**
Update Tracker Protocol page for clarity, removing deprecated (or never implemented) features.

Also adds example requests for Self Describing, Custom Context, Page View and Page Ping events.

### CODE REVIEW

File: docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/example-requests/index.md [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+    }
+  }
+}
+```
+
+<details>
+  <summary>cURL example</summary>
+  <div>
+    <div>
+      <CodeBlock language="bash">
```
Comment (stanch) [inline docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/example-requests/index.md:80]: Why though? I think ``` should work

File: docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/example-requests/index.md [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+      "sid": "be9520e7-16a5-4d4e-afa1-8e269f99a1cf",
+      "duid": "85094061-f702-4b62-a46d-20f7226b4741",
+      "url": "https://snowplow.io/",
+      "stm": "1665398554153"
+    }
+  ]
+}
+```
+
+The encoded base64 string above represents the following JSON:
```
Comment (stanch) [inline docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/example-requests/index.md:61]: Above where? Assumes users know what base64 looks like. Is it `ue_px`? I would state that explicitly

File: docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/example-requests/index.md [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+title: "Example Tracker Requests"
+date: "2022-10-10"
+sidebar_position: 100
+---
+
+# Example Tracker Requests
+
+Below are a number of example Tracker Protocol requests. All examples are POST requests with a JSON body.
+
+All requests are sent to `com.snowplowanalytics.snowplow/tp2` unless you have specified a custom POST path ([via Snowplow Console](/docs/using-the-snowplow-console/accessing-collector-configuration/index.md) or [Collector Configuration](/docs/pipeline-components-and-applications/stream-collector/configure/index.md)).
```
Comment (stanch) [inline docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/example-requests/index.md:11]:
```suggestion
All requests are sent to `https://<your-collector-host>/com.snowplowanalytics.snowplow/tp2` unless you have specified a custom POST path ([via Snowplow Console](/docs/using-the-snowplow-console/accessing-collector-configuration/index.md) or [Collector Configuration](/docs/pipeline-components-and-applications/stream-collector/configure/index.md)).
```

File: docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md [hunk 4]
```diff
@@ -5,50 +5,63 @@ date: "2020-02-26"
 sidebar_position: 80
 ---
 
-## Overview
+# Tracker Protocol Overview
 
-Snowplow trackers fire _events_, which are either `GET` or `POST` requests of a Snowplow collector, whenever an event on a website or application takes place. By appending events' parameters and values to the end of those `GET` requests or adding them into the body of `POST` messages, trackers can pass data into the collectors, for processing by Snowplow.
+Snowplow trackers fire _events_, which are either `GET` or `POST` requests to a Snowplow collector. By appending an events parameters to these requests, trackers can pass data into the collector for processing by Snowplow.
```
Comment (stanch) [inline docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md:10]: Something is off here. Maybe “By appending parameters...”?

File: docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md [hunk 5]
```diff
@@ -5,50 +5,63 @@ date: "2020-02-26"
...
-
-In the first part of this guide, we cover the parameters in the Snowplow tracker protocol that are common across different event types. In the second part, we document the parameters that are relevant for specific events that are recognised in the Snowplow event model. Please note: this model is evolving over time as we incorporate more events and grow the set of fields associated with each of the standard events. In all cases, we do our best to ensure that any changes are backwards compatible. (So we are happy adding new parameters, but do not remove parameters once they have been incorporated.)
+This documentation should be used by anyone who would like to build their own tracker or who would like to understand the tracker payload in more detail. By utilising the parameters documented here, the author of a new tracker can be confident that his / her tracker will work with the rest of the Snowplow stack.
```
Comment (stanch) [inline docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md:14]:
```suggestion
This page is for you if you want to understand the tracker payload in more detail, and especially if you are building your own tracker. In the latter case, utilizing the parameters documented here will ensure that your tracker works with the rest of the Snowplow stack.
```

File: docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md [hunk 6]
```diff
@@ -5,50 +5,63 @@ date: "2020-02-26"
...
 
 Each parameter of the Snowplow Tracker Protocol is described in the tables below, whose columns are:
```
Comment (stanch) [inline docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md:16]: Maybe just remove this? It adds 0 information

File: docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md [hunk 7]
```diff
@@ -5,50 +5,63 @@ date: "2020-02-26"
...
-| `tna`         | `name_tracker`              | text     | The tracker namespace                       | Yes              | `tracker_1`             |
-| `evn`         | `event_vendor` (deprecated) | text     | The company who developed the event model   | deprecated       | `com.snowplowanalytics` |
-| `aid`         | `app_id`                    | text     | Unique identifier for website / application | Yes              | `angry-birds-android`   |
-| `p`           | `platform`                  | text     | The platform the app runs on                | Yes              | `web`, `mob`, `app`     |
+## Common Parameters
 
-The tracker namespace parameter is used to distinguish between different trackers. The name can be any string that _does not_ contain a colon or semi-colon character. Tracker namespacing allows you to run multiple trackers, pinging to different collectors.
+### Event parameters
+
+| **Parameter** | **Maps to**           | **Type** | **Description**  | **Example values**                     |
```
Comment (stanch) [inline docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md:34]: Change “Maps to” to “Table column” or something like that

File: docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md [hunk 8]
```diff
@@ -63,509 +76,222 @@ As a Snowplow user, you can define application IDs for each of your different di
...
-  "schema": "iglu:com.my_company/user/jsonschema/1-0-0",
-  "data": {
-    "fb_uid": "9999xyz"
-  }
-}
-```
+:::
 
-All custom contexts to be attached to an event will be wrapped in an array by the user and passed to the tracker, which will wrap them in a self-describing JSON:
+All context to be attached to an event will be wrapped in an array by the user and passed to the tracker, which will wrap them in a self-describing JSON:
```
Comment (stanch) [inline docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md:294]:
```suggestion
All entities attached to an event will be wrapped in an array by the user and passed to the tracker, which will wrap them in a self-describing JSON:
```

**Review by stanch — CHANGES_REQUESTED** [review]

File: docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/example-requests/index.md (hunk 1 above)
Comment (paulboocock) [reply docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/example-requests/index.md:80]: I'm having zero fun with these errors, and I'm still getting them even with this, I think I might have gone in the wrong direction though...

> SyntaxError: /Users/paulboocock/snowplow/snowplow.github.io/docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/example-requests/index.md: Unexpected token, expected "}" (71:14)
  69 |       --header 'Content-Type: application/json' \ 
  70 |       --data '{
> 71 |       "schema": "iglu:com.snowplowanalytics.snowplow/payload_data/jsonschema/1-0-4",
     |               ^
  72 |       "data": [
  73 |         {
  74 |           "e": "ue",

**Pull request merged**